auto_submit/lib/service/validation_service.dart

```diff
@@ -53,6 +56,7 @@ class ValidationService {
   }
 
   Revert? revertValidation;
+  ApproverService? approverService;
```

Comment:
You can mark this late since it's instantiated in the constructor

```suggestion
  late ApproverService approverService;
```